changelog/unreleased/issue-3083

```diff
@@ -0,0 +1,19 @@
+Enhancement: Allow usage of deprecated S3 ListObjectsV1 API
+
+Restic didn't work with some S3 API implementations that don't have a
+broken `ListObjectsV2` API endpoint. This concerns mainly Ceph versions
+before v14.2.5 , but may impact other S3 API implementations. When a broken
+server implementation is used, restic prints errors similar to the following:
```

Comment:
Please change to the following to fix grammar and meaning errors:

~~~
Some S3 API implementations, e.g. Ceph before version 14.2.5, has a broken
`ListObjectsV2` implementation which cause problems for restic when using
their API endpoints. When a broken server implementation is used, restic prints
errors similar to the following:
~~~